python/CMakeLists.txt

```diff
@@ -7,14 +7,13 @@ add_subdirectory(doc)
 file(GLOB_RECURSE PYTHON_SRC_FILES
   RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}/*.py)
-set(ARTS_BINARY_DIR ${CMAKE_BINARY_DIR/src})
```

Comment:
Here is the potential bug!  @olemke  or @simonpf , where do you use this?  I think it is a bad idea to overwrite standard CMake variables, but I suspect you need this to link the .so file?